@ecomplus/storefront-components/src/js/TheAccount.js

```diff
@@ -138,18 +159,9 @@ export default {
     ]
     const { favorites } = this.ecomPassport.getCustomer()
     this.favoriteIds = favorites || []
-    if (this.ecomPassport.checkAuthorization()) {
-      this.ecomPassport.requestApi('/orders.json?transactions.type=recurrence&limit=1&fields=_id')
-        .then(({ data }) => {
-          const { result } = data
-          if (result.length) {
-            this.navTabs.push({
-              label: this.i19subscriptions,
-              value: 'subscriptions'
-            })
-          }
-        })
-        .catch(console.error)
-    }
+    this.insertSubscriptionTab()
+    this.ecomPassport.on('login', () => {
+      this.insertSubscriptionTab()
+    })
```

Comment:
Linkando novamente o mesmo exemplo do comentário anterior: https://github.com/ecomplus/storefront/blob/master/%40ecomplus/storefront-components/src/js/PointsApplier.js#L105-L108

Listeners precisam ser destruídos pra não gerar memory leak:

```suggestion
    this.insertSubscriptionTab()
    this.ecomPassport.on('login', this.insertSubscriptionTab)
    this.$once('hook:beforeDestroy', () => {
      this.ecomPassport.off('login', this.insertSubscriptionTab)
    })
```